packages/chain-agnostic-permission/src/adapters/caip-permission-adapter-accounts.ts

```diff
@@ -139,3 +139,71 @@ export const setEthAccounts = (
     ),
   };
 };
+
+/**
+ * Sets the permitted accounts for the given scopes object.
```

Comment:
worth mentioning the caveat that the scope must already exist in the scopesObject?